_posts/2023-02-01-solidity-0.8.18-release-announcement.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.18 Release Announcement'
+date: '2023-02-01'
+author: Solidity Team
+category: Releases
+---
+
+## Introducing the newest version of the Solidity Compiler!
+We are excited to announce the latest release of the Solidity Compiler, [Solidity v0.8.18](https://github.com/ethereum/solidity/releases/tag/v0.8.18). 
+This latest version includes a range of improvements and it also introduces support for the [Paris upgrade](https://blog.ethereum.org/2022/08/24/mainnet-merge-announcement)!
+
+
+### Noteworthy Features
```

Comment:
```suggestion
### Notable New Features
```

In previews release announcements, we have referred to this as "Notable New Features" and then had subtitles for each of the Notable New Features. See here for reference: https://blog.soliditylang.org/2022/06/15/solidity-0.8.15-release-announcement/

If we want to re-name and re-structure how that section was usually done that's fine with me. Just pointing out that we should stick to one format moving forward and I'm personally not sure, if the bullet point list is a good tool for going into things more detailed.

Reply:
If we wanted to structure it more closely to how we did in the past, I'd call the subsections:

`--no-cbor-metadata`Command-Line Flag
Support for Paris Hardfork
Deprecation of SELFDESTRUCT

Reply:
Yes, I agree that sticking to the previous format and having subsections would be nice. Definitely better than the bullet points.